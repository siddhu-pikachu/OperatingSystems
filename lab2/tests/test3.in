# scalar is 0
# ./(executable_name) -d test3.in
# can also check for other options (-a, -s, -m)

0

2 1
1,
2,