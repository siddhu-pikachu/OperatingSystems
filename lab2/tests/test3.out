Cannot divide by zero, really?!
